src/wovnio/wovnphp/API.php

```diff
@@ -50,6 +50,7 @@ public static function translate($store, $headers, $original_content, $request_o
         $computedUrl = self::getUriRepresentation($headers->urlKeepTrailingSlash, $store, $headers->requestLang());
         $data = array(
             'url' => $computedUrl,  // rewrite URL to use source lang's "virtual" url.
+            'browser_url' => $headers->originalUrl,
```

Comment:
`$headers->originalUrl` meas requested URL which can include lang code. So, it is same URL which is shown in browser. (I feel confused about what is original.)

I will approve.